metrics/web/lib/common/presentation/routes/metrics_page_route.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class MetricsPageRoute<T> extends MaterialPageRoute<T> {
```

Comment:
It seems like this class lacks documentation. Also, should we add tests for this class?